CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ All notable changes to this project will be documented in this file.
  * #### Added
  * #### Changed
  * #### Removed
+   * Support for measure_multiple.
```

Comment:
Add more context, this is for clients to know what the hell is going on.

measure_mutliple until issue #444 is addressed

Reply:
Fixed